framework/test/mocha/unit/modules/chain/helpers/config.js

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const config = require('../../../../../../src/modules/chain/helpers/config');
+const {
+	ZSchema,
+} = require('../../../../../../src/controller/helpers/validator');
+
+describe('helpers/config', () => {
+	const env = _.cloneDeep(process.env);
+	const argv = _.cloneDeep(process.argv);
```

Comment:
Prefer using JSON methods over lodash,

```js
	const env = JSON.stringfiy(process.env);
	const argv = JSON.stringfiy(process.argv);
```
and use it in before each like below:
```js
	process.argv = JSON.parse(argv);
	process.env = JSON.parse(env);
```